app/views/dashboard/show.html.erb

```diff
@@ -2,15 +2,23 @@
   <div class="govuk-grid-column-full">
     <h1 class="govuk-heading-l">User dashboard</h1>
     <p class="govuk-body"> You are logged in</p>
-
     <% if @current_user.early_career_teacher? %>
       <h2 class="govuk-heading-l">Your Core Induction Programme</h2>
-      <% if @current_user.course_years.each do |course_year| %>
+      <% @current_user.course_years.each do |course_year| %>
         <p><%= govuk_link_to course_year.title, cip_year_url(course_year) %></p>
-      <% end.empty? %>
+      <% end %>
+      <% if @current_user.course_years.empty? %>
         <p>You have not been enrolled in Core Induction Programme yet!</p>
       <% end %>
     <% end %>
-
+    <%if @current_user.mentor? %>
+      <h2 class="govuk-heading-l">Your list of mentees</h2>
+      <% @current_user.mentor_profile.early_career_teachers.each do |early_career_teacher| %>
+        <p><%= early_career_teacher.full_name %></p>
+      <% end %>
+      <% if @current_user.mentor_profile.early_career_teachers.empty? %>
```

Comment:
@amazimbe am I right in thinking this could be written as `end.empty?` ? Should it be?

Reply:
Not sure why but VSCode indentation goes out of wack when we do end.empty

Reply:
It could be.

Steven's auto formatter doesn't believe it should be though. And delving into depths of VSCode plugins is the last thing I want to do. 

When he gets a ruby mine license we will probably rewrite it.

Reply:
I think line 7 would need it's `if` back. Not important though, was mostly just curious 🙂